include/cantera/transport/GasTransport.h

```diff
@@ -173,14 +156,8 @@ class GasTransport : public Transport
     //! Update the temperature-dependent viscosity terms.
     /**
      * Updates the array of pure species viscosities, and the weighting
-     * functions in the viscosity mixture rule. The flag m_visc_ok is set to true.
-     *
-     * The formula for the weighting function is from Poling et al. @cite poling2001,
```

Comment:
Assuming that the equations are redundant ("wilke rule"?), the citation should be moved to `viscosity()`.

Reply:
Yes, it's the same equation, with slightly different formatting. The citation was already moved to `viscosity()` -- the two formulas there go together. This one on its own is essentially meaningless.